components/navigationRail.tsx

```
@@ -0,0 +1,142 @@
+import { AccountDropdown } from "@/app/(dashboard)/mailboxes/[mailbox_slug]/accountDropdown";
+import { api } from "@/trpc/react";
+import { cn } from "@/lib/utils";
+
+export function NavigationRail() {
+  const pathname = usePathname();
+  const router = useRouter();
+  const [mailboxSlug, setMailboxSlug] = useState<string | null>(null);
+  const { data: mailboxes } = api.mailbox.list.useQuery();
+  const currentMailbox = mailboxSlug
+    ? mailboxes?.find((m) => m.slug === mailboxSlug)
+    : mailboxes?.[0];
```

ESLint

[error] Replace `⏎····?·mailboxes?.find((m)·=>·m.slug·===·mailboxSlug)⏎···` with `·?·mailboxes?.find((m)·=>·m.slug·===·mailboxSlug)`

(prettier/prettier)